graphql-client/src/Data/GraphQL/Error.hs

```diff
@@ -30,13 +33,35 @@ data GraphQLError = GraphQLError
   }
   deriving (Show, Eq, Generic, ToJSON, FromJSON)
 
+displayGraphQLError :: String -> String -> GraphQLError -> String
+displayGraphQLError indentFirst indentRest exception =
+  indentFirst
+    <> "At "
+    <> joinWithCommas (maybe [] (map displayErrorLoc) (locations exception))
+    <> ": "
+    -- Note: If this message contains multiple lines, subsequent lines will not be indented correctly.
+    <> unpack (message exception)
+    <> maybe "" (nextLine "Path: " . joinWithCommas . map valueToString) (path exception)
+    <> maybe "" (nextLine "Extensions: " . valueToString) (extensions exception)
+  where
+    joinWithCommas = unwords . intersperse ", "
```

Comment:
This doesn't seem right; wouldn't it do
```hs
["a", "b"] => "a ,  b"
```
Shouldn't it just be `intercalate ", "`?